docs/getting_started/install.md

```diff
@@ -18,8 +18,8 @@ The `kops` binary is also available via our [releases](https://github.com/kubern
 
 ```shell
 curl -LO https://github.com/kubernetes/kops/releases/download/$(curl -s https://api.github.com/repos/kubernetes/kops/releases/latest | grep tag_name | cut -d '"' -f 4)/kops-linux-amd64
-chmod +x kops-linux-amd64
-sudo mv kops-linux-amd64 /usr/local/bin/kops
+chmod +x kops
+sudo mv kops /usr/local/bin/kops
```

Comment:
Why this change? The linux binary is named `kops-linux-amd64`?

Reply:
@hakman thanks. I just realized that it is because of the way it being download in here & in github readme docs.
`curl -LO` vs `curl -Lo kops`. 

So here, it should be `kops-linux-amd64` same are remote binary name.

(do you think we should unify it in both places?)

Reply:
Let's keep the `-LO` version for now in both places. Thanks for finding this.

Reply:
Actually I looked in the file and there is `curl -Lo kubectl` also. So let's go with `-Lo kops` to keep same style.

Reply:
thanks. I will make it uniform across. Could you PTAL e644b530725caefbf96107da67d79ec601105ad2

Reply:
Thanks. Looks ok. Please squash also the changes.

Reply:
done. thanks.